### PR TITLE
[Feature] Add copy story ID to clipboard at collapsed story card

### PR DESCRIPTION
**Pull request by joaoGabriel55**
From Issue #788 

image

### CODE REVIEW

File: app/assets/javascripts/views/story_view.js [hunk 1]
```diff
@@ -541,34 +542,29 @@ const StoryView = FormView.extend({
...
-        />,
+        <StoryEstimateButtons points={this.model.point_values()} onClick={this.estimate} />,
         estimateButtons
       );
     }
 
+    const copyStoryIdClipboardLink = this.$('[data-story-id-copy-clipboard]').get(0)
+    if(copyStoryIdClipboardLink) {
+      ReactDOM.render(<StoryCopyIdClipboard id={this.id} />, copyStoryIdClipboardLink)
+      new Clipboard('.story-id');
```
Comment (talyssonoc) [inline app/assets/javascripts/views/story_view.js:565]: Can we use it like this instead? https://github.com/Codeminer42/cm42-central/blob/d83c594c78612c9161f9c007844141d7aab7f936/app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryHistoryLocation.js#L30-L37

Comment (joaoGabriel55) [reply app/assets/javascripts/views/story_view.js:565]: I will take a look

Comment (talyssonoc) [inline app/assets/javascripts/views/story_view.js:565]: Do we still need this line if we use that other component?

**Comment by GabrielRMuller** [comment]
changes merged through #906

**Pull request closed without merging**